membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -33,6 +33,9 @@ class AccountController extends LazyLogging {
   lazy val mmaCardAction = NoCacheAction andThen corsCardFilter andThen BackendFromCookieAction
 
   def updateCard[P <: SubscriptionPlan.AnyPlan : SubPlanReads] = mmaCardAction.async { implicit request =>
+
+    // TODO - refactor to use the Zuora-only based lookup, like in AttributeController.pickAttributes - https://trello.com/c/RlESb8jG
```

Comment:
if we really want to do these salesforce removals, maybe we can have a cloudwatch metric and dashboard it! ;)

Reply:
I suppose the metric is "number of high priority trello cards"

Reply:
It's one more piece of functionality that'll work if Salesforce goes down. Which it rarely does!